### PR TITLE
docs(module): add "admission-policy-engine and vm" to faq

### PR DESCRIPTION
**Pull request by hardcoretime**
## Description
Add examples with security policies which are required for virtual machine running.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

### CODE REVIEW

File: docs/FAQ_RU.md [hunk 1]
```diff
@@ -273,3 +273,112 @@ spec:
     - kind: VirtualDisk
       name: win-disk
 ```
+
+## Как запустить виртуальную машину при включенных политиках безопасности Admission-policy-engine
```
Comment (fl64) [inline docs/FAQ_RU.md:277]:
```suggestion
## Использование виртуализации совместно с модулем Admission-policy-engine
```

File: docs/FAQ_RU.md [hunk 2]
```diff
@@ -273,3 +273,112 @@ spec:
...
+
+Перед началом рекомендуется ознакомиться с работой модуля [Admission-policy-engine](https://deckhouse.ru/documentation/v1/modules/015-admission-policy-engine/).
```
Comment (fl64) [inline docs/FAQ_RU.md:279]:
```suggestion
Для начала рекомендуется ознакомиться с настройками модуля [Admission-policy-engine](https://deckhouse.ru/documentation/v1/modules/015-admission-policy-engine/).  
```

File: docs/FAQ_RU.md [hunk 3]
```diff
@@ -273,3 +273,112 @@ spec:
     - kind: VirtualDisk
       name: win-disk
 ```
+
+## Как запустить виртуальную машину при включенных политиках безопасности Admission-policy-engine
+
+Перед началом рекомендуется ознакомиться с работой модуля [Admission-policy-engine](https://deckhouse.ru/documentation/v1/modules/015-admission-policy-engine/).  
+При настройке политик рекомендуется опираться на политики безопасности, которые установленны в Вашей компании.
```
Comment (fl64) [inline docs/FAQ_RU.md:280]:
```suggestion
При настройке политик безопасности рекомендуется руководствоваться политиками безопасности, которые установленны в вашей компании.
```

File: docs/FAQ_RU.md [hunk 4]
```diff
@@ -273,3 +273,112 @@ spec:
...
+  - kubernetes
+```
+2. Настройка новой политики безопасности.  
+Эта политика основана на Baseline и позволяет запускать виртуальные машины в заданном прострастве имён.
+```yaml
+---
+apiVersion: deckhouse.io/v1alpha1
+kind: SecurityPolicy
+metadata:
+  name: virt-launcher-deny
```
Comment (fl64) [inline docs/FAQ_RU.md:305]: Why deny? Mb allow?

**Review by fl64 — CHANGES_REQUESTED** [review]

File: docs/FAQ_RU.md (hunk 4 above)
Comment (hardcoretime) [reply docs/FAQ_RU.md:305]: Like d8-pod-security-baseline-deny-default. I suppose it depends on "enforcementAction" field value.

**Review by fl64 — APPROVED** [review]

**Pull request merged**